modules/enabling-team-sync-oidc.adoc

```diff
@@ -0,0 +1,210 @@
+<6> Required. Adds additional scopes that {productname} uses to communicate with the OIDC provider.
+<7> Required. Whether to allow for team membership to be synced from a backing group in the authentication engine.
+<8> Optional. If enabled, non-superusers can setup team synchronization.
+
+. Restart your {productname} registry. 
+
+[id="setting-up-quay-team-sync"]
+== Setting up your {productname} deployment for team synchronization
+
+. Log in to your {productname} registry via your OIDC provider.
+
+. On the {productname} v2 UI dashboard, click *Create Organization*.
+
+. Enter and Organization name, for example, `test-org`.
+
+. Click the name of the Organization.
+
+. In the navigation pane, click *Teams and membership*. 
+
+. Click *Create new team* and enter a name, for example, `testteam`.
+
+. On the *Create team* pop-up:
+
+.. Optional. Add this team to a repository.
+.. Add a team member, for example, `user1`, by typing in the user's account name.
+.. Add a robot account to this team. This page provides the option to create a robot account.
+
+. Click *Next*.
+
+. On the *Review and Finish* page, review the information that you have provided and click *Review and Finish*. 
+
+. To enable team synchronization for your {productname} OIDC deployment, click *Enable Directory Sync* on the *Teams and membership* page. Note the message in the popup:
++
+[WARNING]
+====
+Please note that once team syncing is enabled, the membership of users who are already part of the team will be revoked. OIDC group will be the single source of truth. This is a non-reversible action. Team's user membership from within Quay will be ready-only.
+====
+
+. In the popup box, enter the name of the group to sync membership with. Then, click *Enable Sync*. 
+
+. You are returned to the *Teams and membership* page. Note that users of this team are temporary removed and are re-added upon logging back in. At this stage, only the robot account is still part of the team.
++
+A banner at the top of the page confirms that the team is synced: 
++
+[source,text]
+----
+This team is synchronized with a group in OIDC and its user membership is therefore read-only.
+----
++
+By clicking the *Directory Synchronization Config* accordion, the OIDC group that your deployment is synchronized with appears.
+
+. Log out of your {productname} registry and continue on to the verification steps.
+
+.Verification
+
+Use the following verification procedure to ensure that `user1` appears as a member of the team.
+
+. Log back in to your {productname} registry.
+
+. Click *Organizations* -> *test-org* -> *test-team* *Teams and memberships*. `user1` now appears as a team member for this team.
+
+.Verification
+
+Use the following procedure to remove the role from `user1` via your OIDC provider, and subsequently remove them from the team on {productname}.
```

Comment:
to remove user1 from the `group` (instead of role. role is specific to keycloak)

Reply:
Updated to Use the following procedure to remove `user1` from a group via your OIDC provider, and subsequently remove them from the team on {productname}.